frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -85,17 +86,7 @@ public boolean sendResponse(ModelWorkerResponse message) {
                     throw new IllegalStateException(
                             "Unexpected job in sendResponse() with 200 status code: " + jobId);
                 }
-                if (jobDone) {
-                    String streamNext =
-                            prediction
-                                    .getHeaders()
-                                    .get(
-                                            org.pytorch.serve.util.messages.RequestInput
-                                                    .TS_STREAM_NEXT);
-                    if (streamNext != null && streamNext.equals("true")) {
-                        jobDone = false;
-                    }
-                }
+
```

Comment:
the left side if condition is used to check if all of the inference requests of a batch complete the streaming response in existing implementation. The result of jobDone is used in WorkerThread do-while condition.

 it will break the existing streaming response if the if-condition is removed.

Reply:
see above